src/__tests__/util.test.ts

```diff
@@ -116,16 +82,8 @@ describe('util', () => {
   })
 
   describe('ensureWorkspacesArray', () => {
-    it('handles bad cases', () => {
-      expect(ensureWorkspacesArray(null as any)).toEqual([])
-      expect(ensureWorkspacesArray({} as any)).toEqual([])
-      expect(ensureWorkspacesArray({ packages: {} } as any)).toEqual([])
-      expect(ensureWorkspacesArray({ packages: [1, 2, '3'] } as any)).toEqual(
-        [],
-      )
-      expect(ensureWorkspacesArray({ packages: ['lol'] } as any)).toEqual([
-        'lol',
-      ])
+    it("handles Yarn's weird format", () => {
+      expect(ensureWorkspacesArray({ packages: [] })).toEqual([])
```

Comment:
How come the existing assertions were removed?

Reply:
I removed the code that they handled. I was going to move it elsewhere, but then forgot, and made some more refactors. I think it's fine to keep it there now.

Reply:
Readded it!